### ISSUE TITLE
dep ensure shows a warning

### ISSUE THREAD

**Issue by ifraixedes** [gorgonia/gorgonia#214]
`dep ensure` shows the following warning

```
Warning: the following project(s) have [[constraint]] stanzas in Gopkg.toml:

  ✗  github.com/chewxy/vecf64
  ✗  golang.org/x/tools
  ✗  gorgonia.org/vecf32

However, these projects are not direct dependencies of the current project:
they are not imported in any .go files, nor are they in the 'required' list in
Gopkg.toml. Dep only applies [[constraint]] rules to direct dependencies, so
these rules will have no effect.

Either import/require packages from these projects so that they become direct
dependencies, or convert each [[constraint]] to an [[override]] to enforce rules
on these projects, if they happen to be transitive dependencies.
```

Should those dependencies converted as suggested or not?

**Comment by chewxy** [comment]
Quite sure those are definitely imported. But yes they should be converted

**Comment by chewxy** [comment]
Thanks for the fix in #215 @ifraixedes
